src/applications/disability-benefits/686c-674/config/chapters/674/student-last-term-information/studentLastTerm.js

```diff
@@ -0,0 +1,180 @@
+import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+import cloneDeep from 'platform/utilities/data/cloneDeep';
+import { TASK_KEYS } from '../../../constants';
+import { genericSchemas } from '../../../generic-schema';
+import { isChapterFieldRequired } from '../../../helpers';
+import { addressSchema } from '../../../address-schema';
+
+import { StudentNameHeader } from '../helpers';
+
+const { date, genericTextInput } = genericSchemas;
+
+const nonMilitaryAddressSchema = obj => {
+  const schema = cloneDeep(obj);
+  delete schema.properties['view:livesOnMilitaryBase'];
+  delete schema.properties['view:livesOnMilitaryBaseInfo'];
+  return schema;
+};
+
+const addressSchemaObj = nonMilitaryAddressSchema(addressSchema);
```

Comment:
The address on this page doesn't require a military address option, so the two view fields above get removed.